src/request.ts

```diff
@@ -73,6 +76,11 @@ const attachErrorDetails = (err: unknown) => {
   throw err
 }
 
+export const addVersionToUrl = (url: string, version: Version = "v3"): string => {
+  if (url.includes("identity")) return url
```

Comment:
Is it safer to explicitly add the version if the url contains `api` or whatever api-gateway is exposed as?

Reply:
Both urls will always have either `api` or `identity` in so, i suppose it doesn't really make a difference which way we do it?